src/main/java/wooteco/subway/domain/Sections.java

```diff
@@ -0,0 +1,158 @@
+package wooteco.subway.domain;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import wooteco.subway.exception.section.DuplicatedSectionException;
+import wooteco.subway.exception.section.InvalidSectionException;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class Sections {
+
+    private static final int FIRST_ELEMENT = 0;
+    private final List<Section> sections;
+
+    public static Sections create(Section... sections) {
+        return create(new ArrayList<>(Arrays.asList(sections)));
+    }
+
+    public static Sections create(List<Section> sections) {
+        return new Sections(sections);
+    }
+
+    public List<Station> asStations() {
+        LinkedList<Station> sortedStation = new LinkedList<>();
+        final Section pivotSection = sections.get(0);
+        sortPreviousSections(sortedStation, pivotSection);
+        sortFollowingSections(sortedStation, pivotSection);
+        return sortedStation;
+    }
+
+    private void sortPreviousSections(LinkedList<Station> sortedSections, Section section) {
```

Comment:
```suggestion
    private void sortPreviousSections(List<Station> sortedSections, Section section) {
```
다른 메서드도 LinkedList -> List로 변경해주세요

Reply:
그리고 `List<Station>` 타입인데 왜 Sections 인가요?
역을 순서대로 정렬하는 로직을 의도한게 맞는거죠?